index.html

```diff
@@ -1,317 +1,304 @@
-              Get my resume
-            </button>
-          </div>
-        </div>
-  
-        <ul class="accordion">
-          <li>
-            <div class="accordion-container">
-              <a class="skill--description-link" href="#"
-                ><span class="skill--description">Languages</span></a
-              >
-              <ion-icon
-                class="chevron-icons"
-                name="chevron-down-sharp"
-              ></ion-icon>
+            <hr />
+            <div class="skills-container">
+                <a class="skills--title-link" href="#">
+                    <span class="skills--title">Skills</span></a>
+                <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
             </div>
-            <ul>
-              <li class="skill">
-                <img class="skill-image" src="images/skill_1.png" alt="" />
-                <span class="skill-name">Javascript</span>
-              </li>
-              <li class="skill">
-                <img class="skill-image" src="images/skill_2.png" alt="" />
-                <span class="skill-name">HTML</span>
-              </li>
-              <li class="skill">
-                <img class="skill-image" src="images/skill_3.png" alt="" />
-                <span class="skill-name">CSS</span>
-              </li>
-            </ul>
-          </li>
-        </ul>
-  
-        <div class="frameworks-container">
-          <a class="framework--title-link" href="#">
-            <span class="framework--title">Frameworks</span></a
-          >
-          <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
-        </div>
-        <hr />
-        <div class="skills-container">
-          <a class="skills--title-link" href="#">
-            <span class="skills--title">Skills</span></a
-          >
-          <ion-icon class="chevron-icons" name="chevron-forward-sharp"></ion-icon>
-        </div>
-      </section>      
-    </main>      
-    </body>
-  </html>
-  
\ No newline at end of file
+        </section>
+
+        <section class="contact" id="contact">
+            <div class="inner-div">
+                <h2>Contact me</h2>
+                <p id="inner-div">
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it
+                </p>
+            </div>
+            <form action="https://formspree.io/html" id="form" class="form" method="post">
+                <ul class="inner-list">
+                    <li>
+                        <input type="text" class="form-control" placeholder="Harrison" name="name" id="name" maxlength="30" required />
+                    </li>
+                    <li>
+                        <input type="email" class="form-control" placeholder="harry@gmail.com" name="email" id="email" pattern="email" required />
```

Comment:
- [Optional ] Using `type="email"` does the basic job already in modern browsers. So using `pattern="email"` attribute just overrides the default behavior, So I recommend you to remove the pattern attribute.